src/CommandingOfficers/DocLight.java

```diff
@@ -0,0 +1,104 @@
+package CommandingOfficers;
+
+import Engine.GameScenario;
+import Engine.Combat.StrikeParams.BattleParams;
+import Engine.GameEvents.GameEventQueue;
+import UI.UIUtils;
+import Units.*;
+
+public class DocLight extends RuinedCommander
+{
+  private static final long serialVersionUID = 1L;
+
+  private static final CommanderInfo coInfo = new instantiator();
+  private static class instantiator extends CommanderInfo
+  {
+    private static final long serialVersionUID = 1L;
+    public instantiator()
+    {
+      super("Doctor Light", UIUtils.MISC);
+      infoPages.add(new InfoPage(
+            "Even here, it is not safe. Even this grave has been defaced.\n"
+          + "Someone has written on this stone: 'Hope rides alone'\n"));
+      infoPages.add(new InfoPage(
+          "Base Zone: 0\n" +
+          "Can deploy your COU only twice, with different stats each time (Protoman is first)\n" +
+          "Protoman Boost: +"+BLUES_POWER+"/"+BLUES_POWER+" stats and +0/"+BLUES_SHIELD+" vs indirect attack ("+(BLUES_POWER+130)+"/"+(BLUES_POWER+BLUES_SHIELD+130)+" vs indirects)\n" +
```

Comment:
Admittedly I'm not terribly well versed in Megaman lore, but why prefix Protoman's variables with "BLUES"?

Reply:
Also the wording here isn't very clear: +30/30 makes sense, +140 defense if attacked by an indirect, but then also a boost for attacking indirects, with an even bigger defensive boost (which isn't needed because indirects don't counter)?

Reply:
It's his original name. (Megaman was originally called Rockman, or sometimes just Rock... which makes Roll's name make a little more sense)

Reply:
I just didn't write out the total stats for both cases, for Protoman, since defense matches attack for direct combat cases.
And nah, he's just super duper tough vs ranged attacks. I don't think the bonus applies to Anti-Tank (counter)attacks at 1 range.

Reply:
Sure, I know Megaman was Rockman, but Protoman is Blues? He's very not blue at all.

> And nah, he's just super duper tough vs ranged attacks

Ok, but are we talking +140 defense, or +300? It seems to claim both.

Reply:
Megaman = Rock
Megaman's sister = Roll
Megaman's older brother = Blues
It's a music joke.

It is an extra 140 on top of the rest of his stats.
+30 from generic DoR stuff (base zone boost+veterancy)
+30 from his normal stats
+140 vs ranged
So that's 140 + 30 + 30 = 200 total bonus
So his final stats when being shot at by arty are 160/300

Reply:
> It's a music joke

... Oh. Of course it is. I'm sure I knew that at one point too. Sigh.

I think DoR just lists the CO-specific boost in the CO power description, and leaves you to apply other bonuses separately. This may be a case where adding partial information creates more confusion until you explain where everything comes from. I think I'd either remove the totals or clarify that along "with Zone and veterancy boosts, total is blah". Or run it by the Discord folks as your representative audience to see what is clear to them.

Reply:
> I think DoR just lists the CO-specific boost in the CO power description

You are correct.
It lists the set of boosted units, and then 1 star for each CO-specific +10 in each of attack and defense.
Adding a generic +10/10 gets you the zone boost.
Adding vet stats gets you the final D2D picture ~sans comm towers~

The only CO that boosts attack/defense on power is Waylon, and no specifics on the number is given.

> This may be a case where adding partial information creates more confusion until you explain where everything comes from

I mean, I have the big ol' blurb at the bottom that gives you the source of everything.
![image](https://github.com/ThislsAUsername/DefendPeace/assets/8674241/acc78d31-9346-43c1-8af5-790b32fafe46)

Reply:
FWIW, a lot of the reason I didn't give totals for Brenner is that he has two most-relevant totals, and another two niche ones.
110/130 base zone
130/150 COU/vet